index.js

```diff
@@ -83,7 +84,9 @@ GarageCmdAccessory.prototype.getState = function(callback) {
       if (state === 'STOPPED' && accessory.ignoreErrors) {
         state = 'CLOSED';
       }
-      accessory.log('State of ' + accessory.name + ' is: ' + state);
+      if(accessory.log_polling)
+        accessory.log('State of ' + accessory.name + ' is: ' + state);
```

Comment:
Can you please wrap this in { } even though it is only one line. Keeps the code organized.